modules/photons_app/helpers.py

```diff
@@ -110,6 +110,12 @@ def clone(self):
     def __repr__(self):
         return f"<Firmware {self.major},{self.minor}:{self.build},{self.install}>"
 
+    def __iter__(self):
+        yield from (self.major, self.minor)
+
+    def __str__(self):
+        return f"{self.major}.{self.minor}"
```

Comment:
```suggestion
        return f"{self.major},{self.minor}"
```

Reply:
This will break existing tests and Interactor.

Reply:
fix the tests :)

I want to move to a place where firmware is represented with a comma instead of a dot. Because too many people think 2.8 and 2.80 are the same and it's infuriating

Reply:
If you want to break it, go ahead: it's yours to break. I'm just not prepared to break any application that uses Photons' device finder to filter on capability without warning and a planned deprecation process.